extensions/amp-twitter/1.0/component.js

```diff
@@ -50,6 +65,32 @@ function TwitterWithRef(
     },
     [requestResize]
   );
+  const options = useMemo(
+    () => ({
+      cards,
+      conversation,
+      limit,
+      momentid,
+      timelineScreenName,
+      timelineSourceType,
+      timelineUserId,
+      tweetLimit,
+      tweetid,
+      ...optionsProps,
+    }),
+    [
+      cards,
+      conversation,
+      limit,
+      momentid,
+      optionsProps,
```

Comment:
Is `optionsProps` an object here? I don't see it used in the storybook, but if that's ever passed as `<Twitter optionsProps={{ x }} />` it'd cause this memo to bloat/bypass.

Reply:
Yes, it's an object that's a catch-all for any additional options that aren't supported in the `props` here. 

I originally used `optionsProp` instead of enumerating the full options listed in the [Twitter API documentation](https://developer.twitter.com/en/docs/twitter-for-websites/javascript-api/guides/scripting-factory-functions) directly, given that their supported API could go out of sync with ours after some time without us being able to catch it proactively, and `optionsProp` was a good way that developers could still configure something that would work if that happened.

Can anything be done to avoid bloat if someone passed in `optionsProps` as you described? Otherwise, it may be worth trying to replace it or at least including a note in the documentation for this component.

Reply:
I'll merge this PR for now and follow up with the prop and documentation changes.

Reply:
Whoops, just saw this. The fix could just be documentation recommending a pattern like this:

```js
const TWITTER_OPTIONS = {
  // make sure to define these once globally!
};

function MyComponent() {
    // etc
  return (
    <Twitter optionsProps={TWITTER_OPTIONS} />
  );
}
```